data_board/common/lithium_internal.h

```diff
@@ -1,36 +1,40 @@
-//Configuration Commands
-#define NO_OP_COMMAND           0x01
-#define RESET_SYSTEM            0x02
-#define TRANSMIT_DATA           0x03
-#define RECEIVE_DATA            0x04
-#define GET_TRANSCEIVER_CONFIG  0x05
-#define SET_TRANSCEIVER_CONFIG  0x06
-#define TELEMETRY_QUERY         0x07
-#define WRITE_FLASH             0x08
-#define RF_CONFIG               0x09
-#define BEACON_DATA             0x10
-#define BEACON_CONFIG           0x11
-#define READ_FIRMWARE_REVISION  0x12
-#define WRITE_OVER_AIR_KEY      0x13
-#define FIRMWARE_UPDATE         0x14
-#define FIRMWARE_PACKET         0x15
-#define FAST_PA_SET             0x20
+/**
+ * Packet intrinsics
+ */
 
-#define BAUD_RATE_9600      0
-#define BAUD_RATE_19200     1
-#define BAUD_RATE_38400     2
-#define BAUD_RATE_76800     3
-#define BAUD_RATE_115200    4
+// First packet header magic byte
+#define SYNC_1 'H'
+// Second packet header magic byte
+#define SYNC_2 'e'
+// The command type of an I-Message
+#define I_MESSAGE_TYPE 0x10
+// The command type of an O-Message
+#define O_MESSAGE_TYPE 0x20
+// The payload length of an ACK O-Message
+#define ACK_LENGTH    0x0a0a
+// The payload length of a NACK O-Message
+#define NACK_LENGTH   0xffff
 
-#define RF_BAUD_RATE_1200   0
-#define RF_BAUD_RATE_9600   1
-#define RF_BAUD_RATE_19200  2
-#define RF_BAUD_RATE_38400  3
+/**
+ * Upper-bound packet sizes
+ */
 
-#define RF_MODULATION_GFSK  0
-#define RF_MODULATION_AFSK  1
-#define RF_MODULATION_BPSK  2
+// Length of the sync bytes
+#define SYNC_BYTES_LENGTH   2
+// Length of the checksumed command and payload length header bytes
+#define HEADER_DATA_LENGTH  4
+// Length of the header/payload checksum bytes
+#define CHECKSUM_LENGTH     2
+// Length of all of the header bytes
+#define HEADER_LENGTH       SYNC_BYTES_LENGTH + HEADER_DATA_LENGTH + CHECKSUM_LENGTH
```

Comment:
This should be wrapped in parenthesis, otherwise things like `HEADER_LENGTH * 4` will misbehave.